components/builder-core/src/file_walker.rs

```diff
@@ -13,20 +13,14 @@
 // limitations under the License.
 
 use hab_core::package::{FromArchive, PackageArchive};
```

Comment:
consider removing this file entirely?

Reply:
From what I can tell, this file isn't used at all.  Delete it.